rust/sealevel/client/src/context.rs

```diff
@@ -199,15 +293,19 @@ impl<'ctx, 'rpc> TxnBuilder<'ctx, 'rpc> {
     pub(crate) fn send_with_pubkey_signer(
         self,
         pubkey: &Pubkey,
+        instructions_path: Option<PathBuf>,
+        chain_name: Option<String>,
     ) -> Option<EncodedConfirmedTransactionWithStatusMeta> {
         let signer = self.ctx.signer_for_pubkey(pubkey);
-        self.send(&[signer.as_deref()], pubkey)
+        self.send(&[signer.as_deref()], pubkey, instructions_path, chain_name)
     }
 
     pub(crate) fn send(
         self,
         maybe_signers: &[Option<&dyn Signer>],
         payer: &Pubkey,
+        instructions_path: Option<PathBuf>,
```

Comment:
a slightly nicer interface to not need `instructions_path` and `chain_name` to be passed as params to all these functions could be to add them as fields of the `TxnBuilder` interface, similar to what we do with the client and `with_client`. Would probably prefer this